docs/model/introduction-to-model-architecture.md

```diff
@@ -1,21 +1,21 @@
-# Basics About Model Architecture
+# Introduction to Model Architecture
 
-In this article you will learn about [entities](#entity), [facades](#facade), [repositories](#repository) and their mutual relations.
+In this article you will learn about model, its dependencies, [entities](#entity), [facades](#facade), [repositories](#repository) and their mutual relations.
 
-## Basics about the model
+## Definition of a model
+Definition of model is adopted from [DDD](https://stackoverflow.com/questions/1222392/can-someone-explain-domain-driven-design-ddd-in-plain-english-please/1222488#1222488) definition of a model. Model is a system of abstractions that describes selected aspect of a domain.
```

Comment:
I would recommend to add some info, that in project we are also using the term "domain" for something else. For example in Rules of model at the end there is "MultidomainEnityClassProvider" and it is about different "domain".